cmd/helm-operator/main.go

```diff
@@ -155,6 +156,12 @@ func main() {
 		os.Exit(1)
 	}
 	fmt.Printf("%#v", gitRemoteConfigCh)
+	gitRemoteReleaseCh, err := git.NewGitRemoteConfig(*gitURL, *gitBranch, *gitChartsPath)
```

Comment:
There is a card for improving main.go. Will be done in due time.

Reply:
I agree that writing new code is a great opportunity for cleaning up old code. Provided it is a minor change otherwise I prefer to do an improvement separately as there is a danger of an implementation bloat. As Sam and you want it done as part of this ticket, I shall provide improvements to main.go here.

A repo is needed to do a Chart install/upgrade.

Reply:
> A repo is needed to do a Chart install/upgrade.

Yes, but two such clones of the same repo already exist. Why must a third be added, and also why is a second needed?